### ISSUE TITLE
[FEATURE] Pop-ups (not notifications)!

### ISSUE THREAD

**Issue by Tzahi12345** [Tzahi12345/YoutubeDL-Material#815]
**Is your feature request related to a problem? Please describe.**
Notifications will be implemented in v4.3.1, but they don't capture all use cases, especially where user attention is more needed. Here are a couple examples where more intrusive pop-ups would be ideal:
* There is an existing concurrent stream that you can join
* youtube-dl binary changed and server needs a restart
* 403 errors have occurred (maybe this can be a normal notification)

**Describe the solution you'd like**
A pop-up that can appear in the top-right corner that demands more immediate attention. It should have actions and be dismissable.

To summarize, we can have two schemes of providing information to a user:
* Notifications, not time-sensitive
* Pop-ups, time-sensitive

Let me know y'all thoughts!
